mteb/abstasks/languages.py

```diff
@@ -14,3 +17,54 @@
 
 with path_to_lang_scripts.open("r") as f:
     ISO_TO_SCRIPT = json.load(f)
+
+
+@dataclass
+class LangScriptFilter:
```

Comment:
a utility class for filtering language codes. Previously within abstasks, moved out as I needed it for the results object.

I also believe this implementation is notably more clear.